lib/tasks/impersonation_admin_user.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+namespace :impersonation_admin_user do
+  task :update, %i[username domain] => :environment do |_task, args|
+    new_username = args[:username]
+    new_domain = args[:domain]
+    User
+      .where(username: '3scaleadmin')
+      .joins(:account).references(:account)
+      .update_all([
+                    "username = :username, email = CONCAT(:username, '+', accounts.self_domain, '@', :domain)",
```

Comment:
Needs to test it with Oracle. `CONCAT` does not exist there

Reply:
Ok, I will check 👍 On the Internet I've seen CONCAT used on Oracle, which I dunno if it's true or not, so I am running Oracle tests to figure it out 😄 If it doesn't work, I will fix it 😉